restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/http/AccountsEndpointTest.java

```diff
@@ -191,6 +191,27 @@ public void testCreateAccount() {
         assertEquals(createdAuthToken, createAccountResponse.get("auth_token").getAsString());
     }
 
+    @Test @Category(FeatureExpTests.class)
+    public void testCreateAccountWithInvalidChars() {
+
+        String username1 = "my?account@company.com";
+        String username2 = "my@account@company.com";
+        String username3 = "my=account@company.com";
+        String password = "RestCom1233@";
+
+        JsonObject createAccountResponse = RestcommAccountsTool.getInstance().createAccount(deploymentUrl.toString(),
```

Comment:
if we use "createAccountResponse" we may assert based on status response

Reply:
@jaimecasero this is a JsonObject and doesn't hold the status response. In the given test case it will be null